README.md

```diff
@@ -197,6 +201,21 @@ There are a number of inconsistencies in how browsers handle form events, valida
 
 ----
 
+<a name="upgrade-to-v2"></a>
+
+## Upgrading from v1.x.x
+
+The main change in `v2` is that classes provided by `o-forms` now conform more strictly to the [BEM naming convention][bem]. All form field classes now follow the element convention, so `o-forms-text` is now `o-forms__text`.
+
+There is also now a main block class of `o-forms` which replaces the previous `o-forms-group` class. Full class changes are below:
+
+- `o-forms-group` becomes `o-forms`
```

Comment:
Would these be better displayed as diffs?
```diff
-<div class="o-forms-group">
+<div class="o-forms">
```

```diff
-	<label class="o-forms-label">Select box</label>
+	<label class="o-forms__label">Select box</label>
-	<select class="o-forms-select">
+	<select class="o-forms__select">
etc...
```

To display diff blocks you do:
```
    ```diff
    - line to remove
    + line to add
    ```
```